### PR TITLE
Significantly improve integration test speed and experience 

### PR DESCRIPTION
**Pull request by akudiyar**
- Switch to reusable image
- Update to testcontainers version with fixed image reusing
- Switch to Centos 8 Dockerfile (Centos 7 containers don't work with
the recent kernels already)
- Enable Cartridge cluster output from container but without extra spam
- Remove erroneous usage of the internal dependency classes

### CODE REVIEW

File: src/test/java/io/tarantool/driver/integration/SharedCartridgeContainer.java [hunk 1]
```diff
@@ -25,8 +28,21 @@ abstract class SharedCartridgeContainer {
...
             container.start();
         }
+
+        // Ryuk kills the containers, but they can clash with each other if started in parallel
+        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
+            logger.info("Stopping Tarantool Cartridge container");
+            container.stop();
+            // Ensure that the container has stopped completely before starting another one
+            try {
+                Thread.sleep(CONTAINER_SHUTDOWN_DELAY);
```
Comment (Elishtar) [inline src/test/java/io/tarantool/driver/integration/SharedCartridgeContainer.java:41]: TimeUnit.MILLISECONDS.sleep(CONTAINER_SHUTDOWN_DELAY);

**Comment by dkasimovskiy** [comment]
@akudiyar do you have a plans to finish this PR or I can close it?

**Comment by akudiyar** [comment]
@dkasimovskiy Yes, we need to merge this eventually, because local testing is near impossible on weak machines like VMs. I'll try to spend some time on this next week

**Comment by akudiyar** [comment]
This PR is obsoleted by #409 and #410, closing it.

**Pull request closed without merging**